Build failure: python3Packages.momepy

### Steps To Reproduce

Steps to reproduce the behavior:
1. build with `nix-build  -I nixpkgs=. -A python311Packages.momepy`

### Build log

One test is failing: only the relevant part
```

=================================== FAILURES ===================================
_______________________ TestDistribution.test_Alignment ________________________

self = <momepy.tests.test_distribution.TestDistribution object at 0x7ffee7ff0c10>

    def test_Alignment(self):
        self.df_buildings["orient"] = mm.Orientation(self.df_buildings).series
        sw = Queen.from_dataframe(self.df_tessellation, ids="uID")
        self.df_buildings["align_sw"] = mm.Alignment(
            self.df_buildings, sw, "uID", self.df_buildings["orient"]
        ).series
        test_value = (
            22.744936872392813
            if Version(shapely.geos_version_string) < Version("3.12")
            else 18.29842494
        )
>       assert self.df_buildings["align_sw"][0] == pytest.approx(test_value)
E       assert 18.299481293674084 == 18.29842494 ± 1.8e-05
E
E         comparison failed
E         Obtained: 18.299481293674084
E         Expected: 18.29842494 ± 1.8e-05

momepy/tests/test_distribution.py:130: AssertionError

```

### Additional context

Add any other context about the problem here.

### Notify maintainers

(note, I've set the project to the geospatial team, which should be enough to have it in the radar maybe?)

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.6.37, NixOS, 24.05 (Uakari), 24.05.20240709.249fbde`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.4`

```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/development/python-modules/momepy/default.nix
+++ b/pkgs/development/python-modules/momepy/default.nix
@@ -19,7 +19,7 @@
 
 buildPythonPackage rec {
   pname = "momepy";
-  version = "0.7.0";
+  version = "0.8.0";
   pyproject = true;
   disabled = pythonOlder "3.10";
 
@@ -27,7 +27,7 @@ buildPythonPackage rec {
     owner = "pysal";
     repo = "momepy";
     rev = "v${version}";
-    hash = "sha256-HVp2a0z+5fbfkNSxnTfZPCgG2SJMlKX/zso14M18mCk=";
+    hash = "sha256-r2iGzk54MsrkYB3Sp9/B1QGKnvqPGUj3MQhk6yqIoXE=";
   };
 
   build-system = [ setuptools-scm ];

